processor/ratelimitprocessor/metadata.yaml

```
@@ -30,6 +30,13 @@ telemetry:
         value_type: int
         monotonic: true
       attributes: ["class", "source_kind", "result"]
+    ratelimit.request.uncompressed.size:
```

```suggestion
    ratelimit.request_uncompressed_size:
```

So it's closer to `ratelimit.request_size`?